test/action_controller/serialization_test.rb

```diff
@@ -79,8 +79,15 @@ def render_object_expired_with_cache_enabled
           generate_cached_serializer(post)
 
           post.title = 'ZOMG a New Post'
-          sleep 0.1
-          render json: post
+
+          expires_in = [
+            PostSerializer._cache_options[:expires_in],
+            CommentSerializer._cache_options[:expires_in],
+          ].max + 200
+
+          Timecop.travel(Time.zone.now + expires_in) do
```

Comment:
I mistyped. Just saying I like the use of the Timecop or Delorean gems when it comes to test things that are time sensitive.